### ISSUE TITLE
errno 98 address already in use

### ISSUE THREAD

**Issue by kjvietti** [penguintutor/web-robot#2]
I loved your instructions and explanations for making Ruby Robot! I was able to follow it to control the robot through SSH (your initial version).

When I try to use the web interface you've created here I get the following error when running the web-robot.py file: errno 98 address already in use.

Any idea what might be causing this and how to remedy it?

**Comment by penguintutor** [comment]
Have you installed a web server on the Raspberry Pi? Perhaps lighttpd or Apache?

If so then they will already be using port 80.

If you edit the webrobot.py file and look for the following entry. 

```
# unique port number - this happens to be the ip address used in the example setup without the dots
#PORT = 10551
# Using standard port as no other webserver running
PORT = 80
```

You can either uncomment out the entry referring to 10551 and comment out the entry referring to port 80 by prefixing it with #, or change the value to another suitable port number.

You will then need to access by including the port number in the web browser

eg. 
http://<ipaddress>:10551

**Comment by kjvietti** [comment]
Yep, I have Apache installed. Thanks for the quick response and thanks again for the awesome project!

-kyle
